src/main/java/util/TokenGenerator.java

```diff
@@ -22,12 +22,12 @@ public String getToken(String username) {
         HttpResponse<JsonNode> jsonResponse  = null;
         try {
             jsonResponse = Unirest.post(TypeLoader.getAppTokenUrl())
-                        .field("grant_type", "password")
-                        .field("client_id", TypeLoader.getAppClientId())
-                        .field("userName", username)
-                        .field("password", TypeLoader.getAppPassword())
-                        .field("resource", TypeLoader.getAppClientId())
-                        .asJson();
+                    .field("grant_type", "client_credentials")
```

Comment:
Quick question, I assume we won't be using the grant_type password anymore ?

Reply:
yeah, it will be client_credentials going forward.